src/main/java/frc/robot/RobotContainer.java

```diff
@@ -181,6 +182,20 @@ public void initializeCommands() {
         }
     }
 
+    /**
+     * This method registers the given command to as a named pathplanner command if the command is present. If not, it registers in its place a placeholder command that outputs a warning to the console
+     * @param string the registered name of the command as shown in the pathplanner auto/path file
+     * @param command the actual command
+     */
+    private void registerPathplannerCommand(String commandName, Command command) {
+        if (Objects.nonNull(command)) {
+            NamedCommands.registerCommand(commandName, command);
+        } else {
+            NamedCommands.registerCommand(commandName,
```

Comment:
This is good, can you also add this line above it so it warns the user on startup that a command is unregistered
`System.err.println(commandName + " is null");